src/Resources/views/_widget.html.twig

```diff
@@ -6,33 +6,9 @@
 
 <div id="qp-container" class="qp-container">
     <div id="qp-descrip" class="qp-descrip">
-        or 4 interest-free payments
-
-        <span id="qp-descrip-min-amount" data-min-amount="{{ minAmount }}" style="{{ amount < minAmount ? '' : 'display: none;' }}">
-            on orders <span class="qp--nowrap">over {{ minAmount|sylius_format_money('USD', sylius.localeCode) }}</span>
-        </span>
-
-        <span id="qp-descrip-max-amount" data-max-amount="{{ maxAmount }}" style="{{ amount > maxAmount ? '' : 'display: none;' }}">
-            on orders <span class="qp--nowrap">up to {{ maxAmount|sylius_format_money('USD', sylius.localeCode) }}</span>
-        </span>
-
-        <span id="qp-descrip__price" style="{{ amount < maxAmount and amount > minAmount ? '' : 'display: none;' }}">
-            of <span class="qp-descrip__price">{{ (amount / 4)|sylius_format_money('USD', sylius.localeCode) }}</span>
-        </span>
+        We support QuadPay
```

Comment:
Isn't above changed too far? Don't we need all the descriptions above?

Reply:
The installment calculations are wrong and the whole thing displays badly. This requires rewriting the functionality